src/components/ebay-menu/index.marko

```diff
@@ -50,6 +51,9 @@ $ var baseClass = input.classPrefix || "menu";
                 var checked = component.isChecked(index);
             }
             <if (item.separator)>
+                $ {
+                    separatorCount += 1;
+                }
                 <hr class=`${baseClass}__separator` role="separator" />
```

Comment:
Is any `js` dependent on or interacting with this `<hr>`? If not, could we not just do a `tabindex="-1"` and an aria hidden on this and fix this without having to fiddle with indexes?

Reply:
I think the issue came from treating these as actual item elements. We loop through all the items and if they are a separator, we render it as so.
My guess is the keyboard part might be triggering from makeup.

Reply:
Seems to be working fine on makeup side (https://makeup.github.io/makeup-js/makeup-menu-button/index.html). Is this Safari?

I am currently re-working all of the makeup keyboard stuff and should hopefully be easier to debug when done.

Reply:
This was Chrome

Reply:
> First I used the mouse to click into the menu, and then I used the keyboard to navigate the items. Then I collapsed the menu and then used the keyboard to gain focus on the menu and hit <ENTER> and then navigated the menu items with arrows once more.

So i would do one thing
Check if this issue is present on current production
If it is, lets file an issue. If not then we need to fix it
At least the first scenario seems correct, when we activate without keyboard it wont show focus on chrome but then it does if we press down and the second item is selected
My guess is the second usecase is just showing where you left off but doesnt show the initial focus.

Reply:
Ok. Yea, it does seem like in the second scenario, it continues from where it left off, which is not ideal. It should really start from the beginning again after the menu is expanded again.

Reply:
Yep, it looks to be a production issue. I created the bug issue: https://github.com/eBay/ebayui-core/issues/1809